samples/server/petstore/jaxrs/jersey1/pom.xml

```diff
@@ -38,7 +38,7 @@
           <stopKey>stopit</stopKey>
           <stopWait>10</stopWait>
           <httpConnector>
-            <port>8080</port>
+            <port>80</port>
```

Comment:
This change makes the server try to start at port 80, which (at least on normal Linux systems) is not permitted for non-privileged processes.